themes/digital.gov/layouts/services/card-service-contact.html

```diff
@@ -58,7 +58,7 @@
   {{- $src := .src -}}
   {{- with .item -}}
   {{ $path := "" }}{{ with .File }}{{ $path = .Path }}{{ else }}{{ $path = .Path }}{{ end }}
-  <div class="card card-service-contact" data-edit-this="{{- $path -}}">
+  <div class="card-service-contact" data-edit-this="{{- $path -}}">
```

Comment:
The idea behind creating a generic `card` class was to work towards a consistent "card" component with variants.

We should still work towards this by either:
1. Adding overriding `card-service-contact` variant styles
2. Removing all uses of the word "card" if it's not one

---

For #1 you'd have to remove the divs with grid classes. The CSS would actually look pretty simple with CSS Grid.

```scss
.card-service-contact {
  // …previous styles
  @include at-media("tablet-lg") {
    // You could also do something like 1fr 3fr 1fr 1fr and it'd make the description column wider.
    grid-template-columns: repeat(4, 1fr);
  }
}
```

Reply:
I've removed the card classes from the services contact template. These are aren't really related to the card component since they are just a simple table row listing, not an actual card component.